_locales/it/messages.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": 
+  {
+    "message": "Tabs Aside",
```

Comment:
I found that translation, I haven't published any extension but, please, the name of the extension should remain "Tabs Aside"

Reply:
Because that is a proper name and should not be translated. 90% of chrome web store extensions aren't with the title translated.
People might think that is a low-quality auto-translated extension.

Reply:
I don't think that it is how it works. But... ok. I think, we can leave